manual/trees/index.md

```diff
@@ -113,13 +132,16 @@ The recusion level of nested splitting is called the "split level"; it can be co
 
 If the split level is set to 0, there is no splitting: all data members are stored in the same branch.
 Data members can also be configured to be non-split as part of the dictionary; see → [I/O]({{ '/manual/io' | relative_url }}).
-The default split level is 1; a split level of 99 means to split all members at any recursion level.
+The default split level of 99 means to split all members at any recursion level.
 
 _Pointers_
 
 While references `X &` are not supported as member types, pointers are.
 If the pointer is non-null, ROOT stores the object pointed to (pointee).
-If multiple pointers point to the same object during one `TTree::Fill()` operation, that pointee will only be stored once; upon reading, all pointers will again point to the same object.
+If multiple pointers within the same branch point to the same object during one `TBranch::Fill()` operation (as invoked by `TTree::Fill()`), that pointee will only be stored once; upon reading, all pointers will again point to the same object.
+
+For the general case, indices into object collections could be persistified instead of pointers.
```

Comment:
```suggestion
For the general case, indices into object collections could be persisted instead of pointers.
```

Reply:
Persistify is actually the transitive verb here, so we think it's ok.